### PR TITLE
added the assets folder and changed the links accordingly

### PR DESCRIPTION
**Pull request by mohit200008**
## Fixes Issue

This PR fixes the following issues :
#example

## Changes proposed

Here comes all the changes proposed through this PR

## Check List (Check all the boxes which are applicable)

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain copyrighted content.
- [ ] The title of my pull request is a short description of the requested changes.

## Screenshots 
Add all the screenshots which support your changes

### CODE REVIEW

**Review by vinzvinci — APPROVED** [review]
Looks good to me 👍 , wait for the maintainer for the final review. Thanks!

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket: Sorry for getting back to you late. Thank you very much for your contribution and time. We would love to see many more contributions from you. I appreciate your efforts :)

**Pull request merged**
